### DISCUSSION TITLE
Create New Data Model Block in Flows

### DISCUSSION THREAD

**Discussion by adussarps** [directus/directus#17950] [Archive]
### Summary

This feature would allow us to programmatically create a new data model in the flow editor.

The api already exists for this feature, all we lack is a nice interface.

### Basic Example

A block create data model; and a payload part.

### Motivation

This feature request comes from a real live example of a "form editor app"; where a user would submit a form proposition, that once approved would become an actual form associated with a table.

### Detailed Design

No RFC provided.

### Requirements List

--

### Drawbacks

--

### Alternatives

Use the api through the code block in the flow.

### Adoption Strategy

Just add the new feature without any breaking change.

**Comment by rijkvanzanten** [comment]
Closing this as the feature request is missing the required information.
